database/connect.ts

```diff
@@ -15,6 +14,7 @@ declare module globalThis {
 function connectOneTimeToDatabase() {
   if (!globalThis.postgresSqlClient) {
     globalThis.postgresSqlClient = postgres({
+      ssl: !!process.env.POSTGRES_URL,
```

Comment:
can we use `Boolean()` here for expressiveness?

Reply:
c39d258